mitiq/pec/pec.py

```diff
@@ -224,14 +239,15 @@ def new_executor(
 
 
 def pec_decorator(
-    representations: List[OperationRepresentation],
+    observable: Optional[Observable] = None,
+    representations: Tuple[OperationRepresentation, ...] = (),
     precision: float = 0.03,
     num_samples: Optional[int] = None,
     force_run_all: bool = True,
     random_state: Optional[Union[int, np.random.RandomState]] = None,
     full_output: bool = False,
 ) -> Callable[
-    [Callable[[Union[QPROGRAM, Any, Any, Any]], float]],
+    [Callable[[Union[QPROGRAM, Any, Any, Any]], QuantumResult]],
     Callable[
         [Union[QPROGRAM, Any, Any, Any]],
         Union[float, Tuple[float, Dict[str, Any]]],
```

Comment:
Unrelated to the new changes but I guess `Union[QPROGRAM, Any, Any, Any]` should be `Union[QPROGRAM, Any]` or maybe just `QPROGRAM`.